sql/expression/common.go

```diff
@@ -52,25 +52,40 @@ func (p *UnaryExpression) IsNullable() bool {
 	return p.Child.IsNullable()
 }
 
-// BinaryExpression is an expression that has two children.
-type BinaryExpression struct {
-	Left  sql.Expression
-	Right sql.Expression
+// BinaryExpressionStub is an expression that has two children.
+type BinaryExpressionStub struct {
```

Comment:
static interface checker to make sure this continues to implement `BinaryExpression`?

Reply:
Can't, it doesn't implement the interface (it's missing WithChildren etc.)